quintilesims/layer0
# 246 log date range api

**What does this pull request do?**
Adds `start` and `end` query params for service & task logs 


**How should this be tested?**
* Smoketests
* Run `l0 service logs --start 'mm/dd hh:mm' --end 'mm/dd hh:mm' api` and make sure logs returned are in the specified time frame. 

**Checklist**
- ~System tests (if applicable)~


closes #246
